composer.json

```diff
@@ -17,20 +17,29 @@
         }
     ],
     "require": {
-        "php": ">=5.3.6",
+        "php": "^7.1",
         "behat/behat": "^3.0",
         "behat/symfony2-extension": "^2.0",
-        "doctrine/data-fixtures": "^1.0"
+        "doctrine/data-fixtures": "^1.0",
+        "doctrine/orm": "^2.6",
+        "symfony/process": "^2.1|^3.0|^4.0"
```

Comment:
isn't it used only for mysql and pgsql backup services? maybe we can move it into suggest + dev sections?